### ISSUE TITLE
Function pointer parameter fails when return type is a primitive

### ISSUE THREAD

**Issue by jeff-hykin** [jeff-hykin/cpp-textmate-grammar#334]
<img width="436" alt="Screen Shot 2019-07-26 at 9 08 30 PM" src="https://user-images.githubusercontent.com/17692058/61988654-b4f97980-afe9-11e9-9c32-fe26f93789dd.png">

```cpp
void func3(Type (*name)(int, int)) {};
void func3(bool (*name)(int, int)) {};
```
I haven't looked into it, its probably a pretty easy fix.

**Comment by matter123** [comment]
This is because `:parameter_or_maybe_value` includes `:builtin_storage_type_initilizer` before `:function_pointer_parameter`.

**Comment by jeff-hykin** [comment]
Thanks 👍 went ahead an merged in a fix. Published with `1.12.21`
